Tools/parametric_model/src/models/aerodynamic_models/linear_wing_model.py

```diff
@@ -0,0 +1,141 @@
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+ * FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ * COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+ * BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS
+ * OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED
+ * AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+"""
+
+__author__ = "Julius Schlapbach"
+__maintainer__ = "Julius Schlapbach"
+__license__ = "BSD 3"
+
+import math
+import numpy as np
+
+from scipy.spatial.transform import Rotation
+from progress.bar import Bar
+
+
+class LinearWingModel():
+    def __init__(self, config_dict, mass):
+        self.air_density = 1.225
+        self.ref_area = config_dict["area"]
+        self.mass = mass
+        self.gravity = 9.81
+
+    def compute_wing_force_features(self, v_airspeed, angle_of_attack, elevator_input):
+        """
+        Model description:
+
+        Compute lift and drag forces in stability axis frame.
+
+        Lift force is modeled as a linear function of the angle of attack and the elevator input (no stall effects)
+        F_Lift = 0.5 * density * area * V_air_xz^2 * (c_L_0 + c_L_alpha * alpha + c_L_delta_e * delta_e)
+
+        Drag force is modeled as a quadratic function of the lift coefficient
+        F_Drag = 0.5 * density * area * V_air_xz^2 * (c_D_0 + 1 / (pi * e * ar) * c_L^2)
+
+        Coefficients for optimization:
+        c_L_0, c_L_alpha, c_L_delta_e, c_D_0, 1/ar
+
+        :param v_airspeed: airspeed in m/s
+        :param angle_of_attack: angle of attack in rad
+
+        :return: regression matrix X for the estimation of x- and z-forces for a single feature
+        """
+
+        # compute dynamic pressure times wing area
+        dyn_pressure = 0.5 * self.air_density * \
+            (v_airspeed[0]**2 + v_airspeed[2]**2)
+        X_wing_aero_frame = np.zeros((3, 5))
+
+        # TODO
+        # # features for drag coefficient computation
+        # cL = 2 * self.mass * (self.gravity * np.cos(flight_path_angle) -
+        #                       angular_acceleration[2]) / (self.air_density * self.ref_area * (v_airspeed[0]**2 + v_airspeed[2]**2))
+        # X_wing_aero_frame[0, 3] = - dyn_pressure
+        # X_wing_aero_frame[0, 4] = - (1 / (np.pi * np.e)) * dyn_pressure * cL**2
+
+        # features for lift coefficient computation
+        X_wing_aero_frame[2, 0] = - dyn_pressure * self.ref_area
+        X_wing_aero_frame[2, 1] = - dyn_pressure * \
+            self.ref_area * angle_of_attack
+        X_wing_aero_frame[2, 2] = - dyn_pressure * \
+            self.ref_area * elevator_input
```

Comment:
Here you are only computing features the first three coefficients: 'c_L_0', 'c_L_alpha', `c_L_delta` and the features for `c_D_0`, `inv_ar` are always zero. Is this expected? If this is the case, we would always have `c_d_0` and `inv_ar` estimating zero

Reply:
Yes, this was done on purpose to first get the lift parameter identification up and running and then implement the drag estimation in a next step. I wouldn't be aware of any issues that might occur, despite possibly senseless results for `c_D_0` and `inv_ar`, which we can ignore for the lift estimation.

Reply:
@sjschlapbach I did a small experiment of removing those from the problem and the estimation results are influenced. So I think there is something related to this

Reply:
I'm gonna resolve this as the entire implementation was changed